pkg/backend/filestate/backend.go

```diff
@@ -99,7 +104,16 @@ func New(d diag.Sink, u string) (Backend, error) {
 		return nil, err
 	}
 
-	bucket, err := blob.OpenBucket(context.TODO(), u)
+	blobmux := blob.DefaultURLMux()
```

Comment:
instead of getting the default and the overriding below, can we just declare tehre here as `var blobmux` and then just set in the `else` block if this isn't gcp.

Reply:
I've done it this way so that if other cloud providers need to override the default they can without modifying a bunch of if/else blocks. Is that ok?